src/metabase/db/data_migrations.clj

```diff
@@ -126,9 +126,9 @@
 ;;; |                                                NEW TYPE SYSTEM                                                 |
 ;;; +----------------------------------------------------------------------------------------------------------------+
 
-;; this is the old new type system. as of v30 these are migrated from a field named special_type to
+;; this is the old new type system. as of v30 these are migrated from a field named semantic_type to
 ;; semantic_type. These migrations target the world before this migration so they are left as is. The migrations talk
```

Comment:
They're migrated from `sematic_type` to... `semantic_type`? I think you mean v39 too. Not v30